recipe/meta.yaml

```diff
@@ -487,7 +504,23 @@ outputs:
         - unset CONDA_BUILD_SYSROOT   # [unix]
         - set "CONDA_BUILD_SYSROOT="  # [win]
         - clang++ -v -c mytest.cxx
-        # on linux, $TRIPLE-clang++ is in this package
+
+  - name: clangxx_impl_{{ target_platform }}
```

Comment:
Why are we building this here? We'll have to build these in `ctng-compilers-feedstock` anyway because only `target_platform == cross_target_platform` is built here right?

Reply:
You added them on this feedstock in https://github.com/conda-forge/clangdev-feedstock/commit/689fdd42928958a9060cceb9c086a8b34b0fd073. Happy to move them to ctng-compiler-activation if that's what you prefer now.

Reply:
Yeah, there's a duplication that I think we should avoid by moving everything to `ctng-compiler-activation-feedstock`.